### PR TITLE
Appointments non-functionals

### PR DESCRIPTION
**Pull request by sergei-maertens**
* Updated the documentation for appointments
* Tweaked the form designer to hide irrelevant form fields when a form is in appointment mode

### CODE REVIEW

File: docs/manual/forms/examples/appointment_form.rst [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+.. _manual_forms_examples_appointment_form_preselect_location:
+
+Voorselectie locatie
+====================
+
+Indien u meerdere afspraaklocaties hebt en deze wil beperken tot één enkele locatie,
+dan kunt u dit op de :ref:`afsprakenconfiguratie <configuration_appointment_index>`
+instellen nadat de koppeling technisch ingesteld is.
+
+.. warning:: Let op - dit schakelt effectief alle overige locaties uit!
```
Comment (CharString) [inline docs/manual/forms/examples/appointment_form.rst:81]: I :heart: nl_be. :)

File: docs/manual/forms/examples/appointment_form.rst [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+
+.. _manual_forms_examples_appointment_form_preselect_location:
+
+Voorselectie locatie
+====================
+
+Indien u meerdere afspraaklocaties hebt en deze wil beperken tot één enkele locatie,
+dan kunt u dit op de :ref:`afsprakenconfiguratie <configuration_appointment_index>`
+instellen nadat de koppeling technisch ingesteld is.
```
Comment (CharString) [inline docs/manual/forms/examples/appointment_form.rst:79]: Wat houdt dat in? De koppeling "technisch instellen". Vereist dat een speciale handeling?

**Review by CharString — APPROVED** [review]
Some non-functional comments  :wink:

File: docs/configuration/appointment/index.rst [hunk 3]
```diff
@@ -1,18 +1,33 @@
...
+
+#. Pick the relevant plugin and click the **Configuratie** link on the right.
+   See below for plugin-specific instructions.
+
+#. Then, navigate to **Configuratie** > **Afspraakconfiguratie**.
+
+#. In the **Afspraakplugin** dropdown, select the plugin you configured and want to use.
+
+#. Click **Opslaan** to save the configuration. Optionally, you can save and edit again
+   to configure a :ref:`pre-selected location <manual_forms_examples_appointment_form_preselect_location>`.
```
[the remark this answers is not in the document]
Comment (sergei-maertens) [reply docs/configuration/appointment/index.rst:25]: I thought the rest of the docs was using this style, but after double checking they are actually in English. I will update it to English strings for consistency.

File: docs/manual/forms/basics.rst [hunk 4]
```diff
@@ -67,17 +70,25 @@ In dit tabblad worden de algemene formulier gegevens weergegeven.
...
   kan deze worden uitgevinkt indien er een slechts beperkt aantal stappen is.
+
 * **Actief**: Vink aan om het formulier beschikbaar te maken voor de
   buitenwereld.
+
 * **Onderhoudsmodus**: Vink aan om het formulier in onderhoudsmodus te zetten.
   Als het formulier in onderhoudsmodus staat, kan het formulier niet gestart
   worden en verschijnt er een melding voor de eindgebruiker. Beheerders kunnen
   het formulier blijven gebruiken.
+
```
[the remark this answers is not in the document]
Comment (sergei-maertens) [reply docs/manual/forms/basics.rst:81]: No, they don't :)

File: docs/manual/forms/examples/appointment_form.rst (hunk 1 above)
Comment (sergei-maertens) [reply docs/manual/forms/examples/appointment_form.rst:81]: what would the nl_nl phrasing be?

File: docs/manual/forms/examples/appointment_form.rst (hunk 2 above)
Comment (sergei-maertens) [reply docs/manual/forms/examples/appointment_form.rst:79]: Basically some admin with (likely) more permissions than a form designer needs to set up the service and configure which appointments plugin is used. That's different from creating a form and making it an appointment form.

**Pull request merged**